<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Cancellation</class>
 <widget class="QDialog" name="Cancellation">
  <property name="windowModality">
   <enum>Qt::WindowModality::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>419</width>
    <height>232</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cancel Booking</string>
  </property>
  <widget class="QPushButton" name="btnCancel">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>120</y>
     <width>75</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>CANCEL</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Token No</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtTokenNo">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>40</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="lblInfo">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>170</y>
     <width>321</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtContactNo">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>70</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Contact No</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
